packages/inventory-insights/src/Insights.js

```diff
@@ -95,10 +95,14 @@ class InventoryRuleList extends Component {
 
     fetchKbaDetails = async (reportsData) => {
         const { filters, searchValue, rows } = this.state;
-        const kbaIds = reportsData.map(report => report.rule.node_id).filter(x => x).join(` OR `);
+        let nodeIdCount = 0;
+        const kbaIds = reportsData.map(report => {
+            report.rule.node_id && nodeIdCount++;
+            return report.rule.node_id;
+        }).filter(x => x).join(` OR `);
         try {
-            const kbaDetailsFetch = (await API.get(`https://access.redhat.com/hydra/rest/search/kcs?q=id:(${kbaIds})&fl=view_uri,id,publishedTitle`,
-                {}, { credentials: 'include' })).data.response.docs;
+            // eslint-disable-next-line max-len
+            const kbaDetailsFetch = (await API.get(`https://access.redhat.com/hydra/rest/search/kcs?q=id:(${kbaIds})&fl=view_uri,id,publishedTitle&rows=${nodeIdCount}`, {}, { credentials: 'include' })).data.response.docs;
```

Comment:
This can be styled into multiple rows
```JS
const kbaDetailsFetch = (await API.get(
  `https://access.redhat.com/hydra/rest/search/kcs?q=id:(${
    kbaIds
  })&fl=view_uri,id,publishedTitle&rows=${
    nodeIdCount
  }`,
  {},
  { credentials: 'include' }
 )).data.response.docs;
```

Reply:
yipes, thats ah lotta rows, but okkkkk